spec/lib/auth0/api/v2/resource_servers_spec.rb

```diff
@@ -73,4 +73,14 @@
       expect { @instance.delete_resource_server(nil) }.to raise_error 'Must supply a valid resource server id'
     end
   end
+
+  context '.patch_resource_server' do
+    it { expect(@instance).to respond_to(:patch_resource_server) }
+    it 'is expected to send patch to /api/v2/resource_servers/1' do
+      expect(@instance).to receive(:patch).with('/api/v2/resource-servers/1', fields: 'fields')
+      expect { @instance.patch_resource_server('1', fields: 'fields') }.not_to raise_error
+    end
+    it { expect { @instance.patch_resource_server('', nil) }.to raise_error 'Must specify a resource server id' }
```

Comment:
Tests are failing on this:

https://travis-ci.org/auth0/ruby-auth0/jobs/488774537#L565

I would just check for `Auth0::InvalidParameter` like the integration tests.

Reply:
Made the requested change.

This actually reminds me of something I've been wondering about: why are `Auth0::MissingClientId` and `Auth0::MissingUserId` the only dedicated exceptions for a specific property? There's no `Auth0::MissingResourceServerId` or `Auth0::MissingConnectionId`, all other resources just use `Auth0::InvalidParameter` for a missing id, same as they do for other missing parameters or for present-but-invalid parameters.